### ISSUE TITLE
VisibleDeprecationWarning

### ISSUE THREAD

**Issue by strictlymomo** [gunnarhaslam/normalize#1]
@gunnarhaslam the script worked well.  FYI, i got this warning message.  I'm running python3 so i had to translate `normalize.py` using 2to3 and format some indentations.

```
Momosukes-MBP-2:normalize momo$ python normalize.py
Normalizing  FKUFF.wav ...

normalize.py:43: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  a = np.empty((nsamples, nchannels, 4), dtype=np.uint8)
```
  
if you want, i can create a Pull Request for support for python3.  if so, please add me as a collaborator.

**Comment by gunnarhaslam** [comment]
beautiful! adding you now

**Comment by gunnarhaslam** [comment]
resolved per #2
